core/src/message/common.rs

```diff
@@ -133,6 +134,18 @@ impl Message {
     pub fn exit_code(&self) -> Option<ExitCode> {
         self.reply.map(|v| v.exit_code())
     }
+
+    /// Consumes self in order to create new `StoredMessage`, which payload
+    /// contains string representation of initial bytes,
+    /// decoded into given type.
+    pub fn with_string_payload<D: Decode + ToString>(self) -> Result<Self, Self> {
```

Comment:
It's not used anywhere ?

Reply:
Here it is in test tools:
https://github.com/gear-tech/gear/blob/4927c9c2c355c71394dfd568371e35ae2270de90/gtest/src/manager.rs#L674-L679
And on-chain:
https://github.com/gear-tech/gear/blob/25b791809af67e64b761ca7220bda47a61ffb73c/pallets/gear/src/internal.rs#L491-L496